pkg/vpcmodel/explainabilityPrint.go

```diff
@@ -185,14 +191,24 @@ func (g *groupedConnLine) explainPerCaseStr(c *VPCConfig, src, dst EndpointElem,
 	case externalRouter == nil && dst.IsExternal():
 		return fmt.Sprintf("%v\tThe dst is external but there is no Floating IP or Public Gateway connecting to public internet\n",
 			noConnection)
-	case ingressBlocking && egressBlocking:
-		return fmt.Sprintf("%vconnection is blocked both by ingress and egress"+tripleNLVars, noConnection,
-			headerPlusPath, details)
-	case ingressBlocking:
-		return fmt.Sprintf("%vconnection is blocked by ingress"+tripleNLVars, noConnection,
-			headerPlusPath, details)
-	case egressBlocking:
-		return fmt.Sprintf("%vconnection is blocked by egress"+tripleNLVars, noConnection,
+	case ingressBlocking || egressBlocking || loadBalancerBlocking:
+		blockedBy := []string{}
+		if loadBalancerBlocking {
+			blockedBy = append(blockedBy, "Load Balancer")
+		}
+		if ingressBlocking {
+			blockedBy = append(blockedBy, "ingress")
+		}
+		if egressBlocking {
+			blockedBy = append(blockedBy, "egress")
+		}
+		l := len(blockedBy)
+		blockedByString := fmt.Sprintf("by %s", blockedBy[0])
+		if l > 1 {
+			blockedByString = fmt.Sprintf("both by %s and %s",
```

Comment:
If it is blocked by all 3 then both is not good...

Reply:
fixed